doc/source/examples/similarity_explanations_imagenet.ipynb

```diff
@@ -631,7 +632,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python [conda env:int-grad-py38]",
+   "display_name": "Python [conda env:int-grad-py38] *",
```

Comment:
This should be changed to either "display_name": "Python 3", or "display_name": "Python 3 (ipykernel)", because having a specific environment name will fail to load a kernel when a user tries to run the notebook.